_episodes/03-lists.md

```diff
@@ -539,4 +539,6 @@ Omitting ending index: ["sep", "oct", "nov", "dec"]
 > {: .solution}
 {: .challenge}
 
+[hadleywickham-tweet]: https://twitter.com/hadleywickham/status/643381054758363136
```

Comment:
Let's use this link on line 211 as well.

Reply:
Great catch !